extra-xfce/catfish/spec

```diff
@@ -1,4 +1,3 @@
-VER=1.4.13
-SRCTBL="https://git.xfce.org/apps/catfish/snapshot/catfish-$VER.tar.gz"
-CHKSUM="sha256::4761a8e4c7fb3df1c0299a8d543d7144e492e88cdcd8fe4b3062ffa43c7c02ad"
-REL=2
+VER=4.15.0
+SRCS="https://archive.xfce.org/src/apps/catfish/${VER%.*}/catfish-$VER.tar.bz2"
+CHKSUMS="whirlpool::858c90d8a56651e1b26c072aa96c4d705d29c43740f88d76f4d7639825a79f93b82f8f38578c8fd82e493345eab2769a908ad7bb7abdf5674c8244f87f970215"
```

Comment:
Why whirlpool?

Reply:
Faster in the low-preformance devices which have no SHA acceleration with more secure.